Use gpgme hg commit 84ad86e8b8ab to support gpgme 1.8.0.  This patch
should be removed once mutt hg commit 84ad86e8b8ab is in an official
release.

--- crypt-gpgme.c.orig	2016-11-27 02:22:51 UTC
+++ crypt-gpgme.c
@@ -4946,7 +4946,7 @@ int smime_gpgme_verify_sender (HEADER *h
   return verify_sender (h, GPGME_PROTOCOL_CMS);
 }
 
-void gpgme_set_sender (const char *sender)
+void mutt_gpgme_set_sender (const char *sender)
 {
   mutt_error ("[setting sender] mailbox: %s\n", sender);
   FREE (&current_sender);
--- crypt-gpgme.h.orig	2016-11-27 02:22:51 UTC
+++ crypt-gpgme.h
@@ -53,6 +53,6 @@ int smime_gpgme_send_menu (HEADER *msg, 
 
 int smime_gpgme_verify_sender (HEADER *h);
 
-void gpgme_set_sender (const char *sender);
+void mutt_gpgme_set_sender (const char *sender);
 
 #endif
--- crypt-mod-pgp-gpgme.c.orig	2016-11-27 02:22:51 UTC
+++ crypt-mod-pgp-gpgme.c
@@ -104,7 +104,7 @@ static BODY *crypt_mod_pgp_make_key_atta
 
 static void crypt_mod_pgp_set_sender (const char *sender)
 {
-  gpgme_set_sender (sender);
+  mutt_gpgme_set_sender (sender);
 }
 
 struct crypt_module_specs crypt_mod_pgp_gpgme =
